tests/infra_test.go

```diff
@@ -71,6 +74,107 @@ var _ = Describe("Infrastructure", func() {
 		return tests.WaitForSuccessfulVMIStart(obj)
 	}
 
+	Describe("Taints and toleration", func() {
+
+		It("should tolerate CriticalAddonsOnly toleration", func() {
+
+			var kvPods *k8sv1.PodList
+			By("finding all nodes that are running kubevirt components", func() {
+				kvPods, err = virtClient.CoreV1().
+					Pods(tests.KubeVirtInstallNamespace).List(metav1.ListOptions{})
+				Expect(err).ShouldNot(HaveOccurred(), "failed listing kubevirt pods")
```

Comment:
We should probably use `ToNot` here instead of `ShouldNot` for consistency.

From a comment in the gomega code:

// Typically Should/ShouldNot are used with Ω and To/ToNot/NotTo are used with Expect
// though this is not enforced.

Reply:
Right. Updated.